tests/models/test_cli.py

```diff
@@ -37,7 +39,7 @@
     RestEndpoint,
     get_safe_port,
     pyfunc_serve_and_score_model,
-    PROTOBUF_REQUIREMENT,
+    PROTOBUF_REQUIREMENT, pyfunc_generate_dockerfile,
```

Comment:
```suggestion
    PROTOBUF_REQUIREMENT,
    pyfunc_generate_dockerfile,
```

to pass the lint check.